pkg/scheduler/framework/framework.go

```diff
@@ -41,7 +41,7 @@ const (
 	// The reasons to use for scheduling decisions.
 	pickedByPolicyReason                  = "picked by scheduling policy"
 	pickFixedInvalidClusterReasonTemplate = "cluster is not eligible for resource placement yet: %s"
-	pickFixedNotFoundClusterReason        = "specified cluster is not found"
+	pickFixedNotFoundClusterReason        = "specified cluster \"%s\" is not found"
```

Comment:
reason is a machine read code, it should not be templated.

Reply:
the reason here is not the one in the condition, more like a message. I think using template should be fine.